qa/rpc-tests/fundrawtransaction.py

```diff
@@ -316,7 +316,7 @@ def run_test(self):
         rawtx   = self.nodes[2].createrawtransaction(inputs, outputs)
         dec_tx  = self.nodes[2].decoderawtransaction(rawtx)
 
-        assert_raises_jsonrpc(-4, "Insufficient funds", self.nodes[2].fundrawtransaction, rawtx)
+        assert_raises_jsonrpc(-4, "Coin Selection Failed", self.nodes[2].fundrawtransaction, rawtx)
```

Comment:
Agree that if there are really insufficient funds it'd be good to reply that. A specific error is better than a non-specific one *if the message is correct*. Better an aspecific error than an incorrect one though. An incorrect "insufficient funds" error could set the caller on a wild goose chase or get them to panic. In that case it's better to return something general which prompts the caller to do their own research.